elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigSchemaValidator.java

```diff
@@ -122,11 +123,36 @@ private static void addEmbeddedMessages(JsonNode root, List<String> list, int de
         }
     }
 
+    private static String extractPointer(JsonNode root, String fieldName) {
+        String pointer = null;
+        if (root.has(fieldName)) {
+            JsonNode node = root.get(fieldName);
+            if (node.has("pointer")) {
+                pointer = node.get("pointer").asText();
+            }
+        }
+
+        return pointer;
+    }
+
     private JsonSchema loadSchema(String resource) {
-        ObjectMapper objectMapper = new ObjectMapper();
-        Reader reader = new InputStreamReader(DynamicConfigHelpers.class.getResourceAsStream(resource));
+
+        Library library = DraftV4LibraryWithElideFormatAttr.getInstance();
+
+        MessageBundle bundle = MessageBundleWithElideMessages.getInstance();
+
+        ValidationConfiguration cfg = ValidationConfiguration.newBuilder()
+                        .setDefaultLibrary("http://my.site/myschema#", library)
+                        .setValidationMessages(bundle)
+                        .freeze();
+
+        JsonSchemaFactory factory = JsonSchemaFactory.newBuilder()
```

Comment:
Any reason not to build this once in the constructor of the DynamicConfigSchemaValidator?

Reply:
updated.